uploader.json

```diff
@@ -1767,6 +1767,13 @@
             "pattern": [],
             "symbolic_name": "usr_journald_conf_d"
         },
+	{
+            "file": "/etc/vdsm/vdsm.conf",
+            "pattern": [
```

Comment:
I cannot see any filters for Specs.vdsm_conf in the Parser, even in your new [PR-1129](https://github.com/RedHatInsights/insights-core/pull/1129), this Ini Configure file shouldn't be filtered. And any unreasonable filter will cause parse Error in the IniConfigFile.

Reply:
Reverting pattern for `vdsm_conf` as empty list